README.md

```diff
@@ -50,11 +54,11 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
 Your Answer:
 
 ```text
-___: Defines the distance between an element's border and adjacent elements' borders.
-___: Inserts a "wall" around an element.
-___: Defines the distance between an element's content and its border.
+_margin__: Defines the distance between an element's border and adjacent elements' borders.
+__border_: Inserts a "wall" around an element.
+__padding_: Defines the distance between an element's content and its border.
 ___: Defines the width of an element.
-___: Overlays a "wall" on top of an element.
+_outline__: Overlays a "wall" on top of an element.
 ___: Defines the distance between the center of an element and the center of the adjacent element.
 ```
```

Comment:
While `margin` is spacing added outside of an element's padding and border (assuming it has some), it does not necessarily extend to another element's borders. That other element may have `margin` as well, or it may not have any `margin`, `border` or `padding` at all. You never know!